package.json

```diff
@@ -33,7 +33,7 @@
   "homepage": "https://github.com/fraserxu/react-chartist",
   "peerDependencies": {
     "chartist": "^0.10.1",
-    "react": "^0.14.9 || ^15.3.0"
+    "react": "^0.14.9 || ^16.0.0"
```

Comment:
This should still work with `15.*.*` too I think.

Reply:
Sure, fixed